library/spdm_common_lib/libspdm_com_opaque_data.c

```diff
@@ -230,3 +230,96 @@ bool libspdm_get_element_from_opaque_data(libspdm_context_t *spdm_context,
 
     return result;
 }
+
+/**
+ *  Process general opaque data check
+ *
+ * @param  data_in_size                  size in bytes of the data_in.
+ * @param  data_in                       A pointer to the buffer to store the opaque data version selection.
+ *
+ * @retval true                           check opaque data successfully
+ * @retval false                          check opaque data failed
+ **/
+bool libspdm_process_general_opaque_data_check(libspdm_context_t *spdm_context,
+                                               size_t data_in_size,
+                                               void *data_in)
+{
+    const spdm_general_opaque_data_table_header_t
+    *spdm_general_opaque_data_table_header;
+    const opaque_element_table_header_t
+    *opaque_element_table_header;
+    uint8_t element_num;
+    uint8_t element_index;
+    uint16_t opaque_element_data_len;
+    size_t data_element_size;
+    size_t current_element_len;
+    size_t total_element_len;
+
+    total_element_len = 0;
+
+    if(data_in_size > SPDM_MAX_OPAQUE_DATA_SIZE)
+    {
+        return false;
+    }
+
+    if(libspdm_get_connection_version (spdm_context) >= SPDM_MESSAGE_VERSION_12) {
```

Comment:
`if(` -> `if (` in multiple places.
I'm surprised uncrustify doesn't catch this. I'll see if it can be enabled.

Reply:
Thanks ,I have changed ` if( ` to  `if  (`